presto-docs/src/main/sphinx/release/release-0.238.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.238
+=============
+
+**Highlights**
+==============
+* Fix case sensitivity issue in SQL function parameters. Function parameters should be treated case-insensitive.
+* Add support to create external function.
+* Add predicate pushdown support for ``DATE``, ``TIMESTAMP``, and ``TIMESTAMP_WITH_TIME_ZONE`` literals.
+
+General Changes
+_______________
+* Fix case sensitivity issue in SQL function parameters. Function parameters should be treated case-insensitive.
+* Add ``target_result_size`` session property to customize data batch sizes being streamed from coordinator.
+* Add optimization to push null filters to the INNER side of equijoins. The optimization can be enabled with ``optimize-nulls-in-joins``.
+* Add sessionProperty override to Presto JDBC URI.
```

Comment:
If session property is intended to be an identifier then we should use double back quotes or just say `session property`